<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cook_form</class>
 <widget class="QMainWindow" name="cook_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель повара. Заказы</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../static/resource.qrc">
    <normaloff>:/img/img/Logotip.png</normaloff>:/img/img/Logotip.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>521</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>-10</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Заказы</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>1231</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="table_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1221</width>
       <height>461</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);</string>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Официант</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество клиентов</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Блюда</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Напитки</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус от официанта</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус от повара</string>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>1231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_auth">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 10pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
      </property>
      <property name="text">
       <string>Выйти на форму авторизации</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_save_status">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>0</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 10pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
      </property>
      <property name="text">
       <string>Сохранить статус</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
